lib/runner.js

```diff
@@ -273,6 +273,10 @@ Runner.prototype.failHook = function(hook, err) {
     hook.originalTitle = hook.originalTitle || hook.title;
     hook.title =
       hook.originalTitle + ' for "' + hook.ctx.currentTest.title + '"';
+  } else {
+    var parent = hook.parent ? hook.parent.title : 'root';
```

Comment:
this is incorrect; your original code in 5ea6e653 was fine.  The test failures in `test/unit/runner.spec.js` are due to bad test fixtures--`parent` is a required property of any `Runnable`.

You can fix the tests by ensuring every newly-instantiated `Test` belongs to a `Suite`, like this:

```js
it('should increment .failures', function() {
  expect(runner.failures, 'to be', 0);
  var test1 = new Test('fail hook 1', noop);
  var test2 = new Test('fail hook 2', noop);
  suite.addTest(test1);
  suite.addTest(test2);
  runner.failHook(test1, {});
  expect(runner.failures, 'to be', 1);
  runner.failHook(test2, {});
  expect(runner.failures, 'to be', 2);
});
```

Reply:
that said, you will need to address the fact that the root suite a title of an empty string. :smile:

Reply:
But suite `title` having an empty string value isn't set in stone, just the default. The root suite _should_ be identified by presence of truthy `root` property.

Reply:
my comment was in reference to how the error message is getting concatenated, anyway.  an empty  `title` would need to be taken into consideration.  

the root suite having an empty string title sounds like one of those things that other code makes assumptions about.